.code-samples.meilisearch.yaml

```diff
@@ -454,3 +454,18 @@ security_guide_list_keys_1: |-
 security_guide_delete_key_1: |-
   client = MeiliSearch::Client.new('http://127.0.0.1:7700', 'masterKey')
   client.delete_key('d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4')
+tenant_token_guide_generate_sdk_1: |-
+  api_key = 'B5KdX2MY2jV6EXfUs6scSfmC'
+  expires_at = Time.new(2025, 12, 20).utc
+  search_rules = {
+    'patient_medical_records' => {
+      'filter' => 'user_id = 1'
+    }
+  }
+
+  token = client.generate_tenant_token(search_rules, api_key: api_key, expires_at: expires_at)
+tenant_token_guide_search_sdk_1: |-
+  front_end_client = MeiliSearch::Client.new('http://127.0.0.1:7700', token)
+
+  medical_records_index = front_end_client.index('patient_medical_records')
+  search = medical_records_index.search('blood test')
```

Comment:
In all code samples, I think we prefer to use the implicit index creation directly like:
```
    medical_records_index('patient_medical_records').search('blood test')
```
And for me, the variable `search =`  is not necessary

Reply:
I think you mean this @alallema 
```
front_end_client. index('patient_medical_records').search('blood test')
```

Reply:
That exactly what I mean sorry

Reply:
oh yeah, I just replaced the original sample with a ruby version 😛